pallets/gear-program/src/tests.rs

```diff
@@ -191,9 +198,11 @@ fn pause_terminated_program_fails() {
 
         run_to_block(2, None);
 
+        let block_number: u32 = System::block_number() as u32;
```

Comment:
The type can be inferred, no need to set it explicitly.
```suggestion
        let block_number = System::block_number() as u32;
```

Reply:
done